### DISCUSSION TITLE
Run colon command from cli at startup, maybe like `vim -c '...'`

### DISCUSSION THREAD

**Discussion by YeungOnion** [helix-editor/helix#8116] [Q&A] [answered]
I'd like to pass a ':set-option' command at startup from the command line. The filetype I'm using is a markdown file, but I don't want those settings on all markdown files otherwise I'd look to see about options based on language.

I thought I might be able to get away with changing a filetype and using the same grammar, but I realized that running commands from the command line may also be an option.

Since I'm asking about setting an option, I'd actually prefer to not have the behavior of `vim -c '...'` since I'd like that processed after configuration, but the general question might be more useful.

**Comment by David-Else** [comment, marked as answer]
You can have a `config.toml` local to a project by putting it under a `.helix` directory in your repository. Its settings will be merged with the configuration directory `config.toml` and the built-in configuration, so if you have the markdown files in question in separate directories with `.helix` directories you should be able to have different settings per directory of markdown files.

**Reply by YeungOnion** [reply]
Thanks! This is dependent on where helix is called from, but that's alright by me.
